### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by bwieckow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+
+### 2. POST /register
+Save user to database, enabling extracting data from GitHub or Google.
+
+|        Input        |
+|   :-------------    |
+| id: number          |
+| username: string    |
+| tokenIssuer: string |
```
Comment (hayuna) [inline README.md:86]: What does it mean token issuer?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+Configure the port by changing `services.api.ports` in __docker-compose.yml__. Port 1234 was used by default so the value is easy to identify and change in the configuration file.
+
+
+## Testing
+Application testing is executed in GitHub Actions in [actions/setup-node](https://github.com/actions/setup-node). You can find configuration files in ```./.github/workflows/```.
+
+
+## API
+This miecroservice has three endpoint which are listed below.
```
Comment (hayuna) [inline README.md]:
```suggestion
This microservice has three endpoint which are listed below.
```

File: README.md (hunk 1 above)
Comment (bwieckow) [reply README.md:86]: This field will contain a name of provider.  Possibly, we will get rid of that if we manage to retrieve this information from token.

File: README.md (hunk 2 above)
Comment (bwieckow) [reply README.md]: Done

**Pull request merged**